### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by kleinermann** [soonick/poMMo#121]
Hi there, 

I think it would be good to tell people to go to www.mydomain.com/install.php in order to start the installation process in the web browser. Otherwise they may get stuck trying to login as admin on the home page....

Currently it only says in the INSTALL document:
1. Visit your poMMo installation in a web browser and complete the online install procedure.

**Comment by soonick** [comment]
When you download poMMo for the first time you are directed automatically to the installation page. Didn't this happen for you?

**Comment by kleinermann** [comment]
No, i uploaded all the files to the server, added the db info into a copy of the config file, and visited the domain i installed poMMo on. Could this have to do with me installing it on a sub-domain?

**Comment by soonick** [comment]
Hello,

I took a look at the INSTALL file and it seems to have old information. I will keep this ticket open to remind me to update it and add something to the wiki.

Thanks for reporting this issue.

**Comment by valerian** [comment]
I was not directed to install.php either but i guessed that i should do it. Other less technical people might get confused.
